### ISSUE TITLE
App Still active After Swipe up from Memory

### ISSUE THREAD

**Issue by rohitsangwan01** [JulianAssmann/flutter_background#49]
**Describe the bug**
Great package Man , but am facing an issue , my app is keeping a connection with bluetooth device in background , and that connection depends on a Stream , so as soon as i close that stream , connection will be lost from Bluetooth device , so it was working fine without this package , when i swipe up app from Memory  , App disconneted from Bluetooth Device , which was fine , but now after using this package , even if i swipe up app , that bluetooth Device remains still connected with my phone , means it still keeping that stream active somehow , even if app is not live in memory , also noticed one thing , when i use app in debug mode with VSCODE , there when we close app and clear from App , Vscode should show "Lost Connection From Device " , but after using this package , even Vscode remains connected , and Hot Reload/Refresh commands Work 

**Smartphone:**

 - Device: One plus 8T
 - OS:11

**Comment by JulianAssmann** [comment]
Mmh interesting, I will investigate the issue when I have the time.

**Comment by rohitsangwan01** [comment]
@JulianAssmann , i think Foreground services keeping the app active even if we swipe up the app from taskbar of mobile , but it calls that Terminate service method in native android, maybe we can have a callback there , so that flutter user can perform some task in that callback , when user swipe up app

**Comment by JulianAssmann** [comment]
Yes, that's a good idea anyway. I was working on callbacks for action buttons anyway, so that is a logical next step. Thanks for the suggestion.
